src/Microsoft.DotNet.Build.Tasks/PackageFiles/buildagainstpackages.targets

```diff
@@ -0,0 +1,181 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <UsingTask TaskName="AddDependenciesToProjectJson" AssemblyFile="$(BuildToolsTaskDir)Microsoft.DotNet.Build.Tasks.dll"/>
+  <UsingTask TaskName="ParsePackageNames" AssemblyFile="$(BuildToolsTaskDir)Microsoft.DotNet.Build.Tasks.dll"/>
+  <UsingTask TaskName="GetPackageNumberFromPackageDrop" AssemblyFile="$(BuildToolsTaskDir)Microsoft.DotNet.Build.Tasks.dll"/>
+
+  <ItemGroup>
+    <_PackagesDrops Include="$(PackagesDrops)" />
+  </ItemGroup>
+  <PropertyGroup>
+    <!-- overridden in repo -->
+    <GeneratedProjectJsonDir Condition="'$(GeneratedProjectJsonDir)' == ''">$(MSBuildThisFileDirectory)generated</GeneratedProjectJsonDir>
```

Comment:
Why are we default this to here? Why not default it to intermediate/generated or something